scanpy/readwrite.py

```diff
@@ -179,12 +179,9 @@ def read_10x_h5(
                     f"Could not find data corresponding to genome '{genome}' in '{filename}'. "
                     f'Available genomes are: {list(adata.var["genome"].unique())}.'
                 )
-            adata = adata[:, list(map(lambda x: x == str(genome), adata.var['genome']))]
+            adata = adata[:, adata.var['genome'] == f"{genome}"]
```

Comment:
```suggestion
            adata = adata[:, adata.var['genome'] == genome]
```

Why the format string?

Reply:
Sorry, wanted to ask the same question: The original code transformed it to a string so I kept it.

Reply:
@ivirshup, do you want me to remove the conversion to string?